test/server/lib/OIDCConfig.ts

```diff
@@ -619,6 +620,53 @@ describe('OIDCConfig', () => {
         extraChecks: checkUserProfile({
           email: 'fake-email2',
           name: 'fake-name2',
+          extra: {},
+        }),
+      },
+      {
+        itMsg: 'should store extra info returned by the SSO provider when the env var is set',
+        session: DEFAULT_SESSION,
+        env: {
+          GRIST_IDP_EXTRA_PROPS: 'extrafield,anotherfield',
```

Comment:
Nitpicking: in order to fully check that all specified properties are pulled from the userinfo, I would check the scenario with the following, where both `extrafield1` and `extrafield2` are present in `userInfo`:
```suggestion
          GRIST_IDP_EXTRA_PROPS: 'extrafield1,extrafield2,anotherfield',
```